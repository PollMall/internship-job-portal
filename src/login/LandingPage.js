import React from 'react';
import { Link } from 'react-router-dom';
import { Grid } from '@material-ui/core';
import LoginForm from './LoginForm';

function LandingPage() {
  return (
    <Grid container direction="column" alignItems="center">
      <h1>Login Form</h1>
      <LoginForm />
      <p>
        Don&apos;t have an account?
        <Link to="/register">
          Register Now
        </Link>
      </p>
    </Grid>
  );
}

export default LandingPage;
