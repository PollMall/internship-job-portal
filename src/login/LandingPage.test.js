import React from 'react';
import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import LandingPage from './LandingPage';

describe('LandingPage test', () => {
  it('should match snapshot', async () => {
    const { container } = render(<LandingPage />, { wrapper: MemoryRouter });
    expect(container).toMatchSnapshot();
  });
});
