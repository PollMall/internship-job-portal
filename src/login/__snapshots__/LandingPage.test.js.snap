// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingPage test should match snapshot 1`] = `
<div>
  <div
    class="MuiGrid-root MuiGrid-container MuiGrid-direction-xs-column MuiGrid-align-items-xs-center"
  >
    <h1>
      Login Form
    </h1>
    <form
      class="makeStyles-root-1"
    >
      <div
        class="MuiFormControl-root MuiTextField-root"
      >
        <label
          class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
          data-shrink="false"
          for="username"
          id="username-label"
        >
          Username
        </label>
        <div
          class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
        >
          <input
            aria-invalid="false"
            class="MuiInputBase-input MuiInput-input"
            id="username"
            name="username"
            type="text"
            value=""
          />
        </div>
      </div>
      <br />
      <div
        class="MuiFormControl-root MuiTextField-root"
      >
        <label
          class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
          data-shrink="false"
          for="password"
          id="password-label"
        >
          Password
        </label>
        <div
          class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
        >
          <input
            aria-invalid="false"
            class="MuiInputBase-input MuiInput-input"
            id="password"
            name="password"
            type="password"
            value=""
          />
        </div>
      </div>
      <br />
      <button
        class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary Mui-disabled Mui-disabled"
        disabled=""
        tabindex="-1"
        type="button"
      >
        <span
          class="MuiButton-label"
        >
          Login
        </span>
      </button>
    </form>
    <p>
      Don't have an account?
      <a
        href="/register"
      >
        Register Now
      </a>
    </p>
  </div>
</div>
`;
